packages/shared/providers.yaml

```diff
@@ -234,6 +234,14 @@ calendly:
         response_type: code
     proxy:
         base_url: https://api.calendly.com
+clari-copilot:
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://rest-api.copilot.clari.com
+        headers:
+            X-Api-Key: ${apiKey}
+            X-Api-Password: ${apiSecret}
```

Comment:
Update to reference this: https://github.com/NangoHQ/nango/pull/1613/files#diff-ea826d3a45056e71af1c312cb0757f978c999d0b4d94afa0e6766afb8226f5efR242

Reply:
Sure @khaliqgant.